### ISSUE TITLE
How do I get a file into CKEditor's URL fieled?

### ISSUE THREAD

**Issue by logofakt** [alexantr/yii2-elfinder#3]
Maybe you answered this question in the last issue but I'm not familiar to russian language so I don't understand your explanation.
With this piece of code I managed to call CKEditor having a "Search server" button right from the URL field:
            <?php
                echo $form->field($model, 'content')->widget(CKEditor::className(), [
                    'options' => ['rows' => 6],
                    'preset' => 'full',
                    'clientOptions' => ['filebrowserBrowseUrl' => yii\helpers\Url::to(['elfinder/ckeditor', 'path' => 'files/wsk'])],
                ]);
            ?>
When I click onto the button a new browser window opens with a "Not Found (#404)" errror. Please tell me which steps I have to accomplish to see the file structure when clicking the button and how the address of a selected file comes into the URL field. I suppose 'elfinder/ckeditor' should lead to a controller action.

**Comment by alexantr** [comment]
> I suppose 'elfinder/ckeditor' should lead to a controller action.
With this extenison you must create all actions manually based on code from readme.

**Comment by logofakt** [comment]
Thanks
